modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -174,18 +174,69 @@ A successful match is made, and `airline_8838` is duly replicated to the target
+Selecting this option determines that if, having been replicated, the document at source is deleted, the replicated copy of the document will _not_ be deleted.
+Conversely, leaving this option unselected or `false` (which are the defaults) replicates deletions that occur at source, meaning that the replicated copy of the document _will_ be deleted.
+
+==== Replication of TTL
+
+Configured through the xref:manage:manage-xdcr/filter-xdcr-replication.adoc#deletion-filters[UI] with the *Remove TTL from replicated items* checkbox; through the xref:cli:cbcli/couchbase-cli-xdcr-replicate.adoc[CLI] with the `reset-expiry` flag; and with the xref:rest-api:rest-xdcr-create-replication.adoc[REST API] with the `filterBypassExpiry` flag.
+Selecting this option determines that the TTL that a document bears at source is _not_ made part of the replicated copy of the document: instead, the TTL of the replicated copy is set to 0.
+Conversely, if this option is not selected or left `false` (which are the defaults), the TTL is made part of the replicated copy of the document, and may thereby determine when the replicated copy of the document expires.
+Note, however, that the TTL applied to the replicated document at the target may be that of either the collection or the bucket in which it resides: for information, see xref:learn:buckets-memory-and-storage/expiration.adoc[Expiration].
 
 === Deletion Filters versus Filter Expressions
 
 By default, any source-document deletion _is_ replicated to the target; resulting in a corresponding target-document deletion.
 Note that such replication is _not_ prevented by the specifying of a filter that is formed with regular and other filtering expressions: such expressions only determine which non-deleted documents are to be replicated.
 Therefore, to ensure that document-deletions are _not_ replicated, _deletion filters_ must specifically be configured.
 
+=== Tombstones and Replication
+
+When a document is deleted or is expired, a tombstone is created.
+Tombstones and their management are described in xref:learn:buckets-memory-and-storage/storage.adoc#tombstones[Tombstones].
+In order to replicate a deletion or an expiration, XDCR must be able to find, on the source, a tombstone that corresponds to the deleted or expired document.
+When the tombstone is located, XDCR generates a corresponding deletion or expiration event, and replicates this to the target.
+
+Tombstones are periodically purged by Couchbase Server.
+If a document has been deleted or expired, and the resulting tombstone has been purged prior to XDCR being able to locate it, no deletion or replication event is replicated.
+This situation might occur if:
+
+* A document is deleted and then immediately recreated, such that the time during which the tombstone existed has been too brief for location of the tombstone by XDCR to occur.
```

Comment:
The tombstone technically still exists... however, the KV engine could decide that the tombstone is superceded by the recreated document, and for the sake of efficiency (deduplication), skip sending the deletion and just send the recreated document instead.

The phrase "tombstone existed has been too brief" would be incorrect and runs contrary to the concept of tombstone purging.

Reply:
I've rewritten this accordingly.